contracts/contracts/governance/Governor.sol

```diff
@@ -84,6 +99,13 @@ contract Governor {
             "Governor::propose: too many actions"
         );
 
+        for (uint256 i = 0; i < signatures.length; i++) {
+            require(
+                keccak256(bytes(signatures[i])) != setPendingAdminSign,
+                "Governor::propose: setPendingAdmin transaction cannot be propsed or queued"
```

Comment:
Typo on proposed

Reply:
Will fix it. The `o` key in my MBP doesn't work always 😅